DigitalLearningSolutions.Web/Views/Support/RequestSupportTicket/RequestAttachment.cshtml

```diff
@@ -0,0 +1,92 @@
+﻿@using DigitalLearningSolutions.Data.Models.Support
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.Support.RequestSupportTicket;
+@using NHSUKViewComponents.Web.ViewComponents
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model RequestAttachmentViewModel
+@{
+    bool errorHasOccurred = false;
+    @if (Model.RequestAttachment == null && !ViewData.ModelState.IsValid)
+    {
+        errorHasOccurred = true;
+    }
+
+}
+@if (errorHasOccurred )
+{
+    <vc:error-summary order-of-property-names="@(new[] { nameof(Model.ImageFiles),nameof(Model.FileSizeError),nameof(Model.FileExtensionError) })" />
+}
+<a class="nhsuk-button nhsuk-button--secondary" role="button" asp-controller="RequestSupportTicket" asp-action="RequestSummary">
```

Comment:
These Back links should use the [NHS Design System back-link component](https://service-manual.nhs.uk/design-system/components/back-link), as per wireframes.

Reply:
This has been implemented and changes are pushed. Please verify.